### PR TITLE
fix error state on input groups

### PR DESCRIPTION
**Pull request by tatelucas**

### CODE REVIEW

**Comment by saylormic** [comment]
:+1:

File: crossroads.net/styles/layout/forms.scss [hunk 1]
```diff
@@ -37,7 +37,14 @@
...
+.input-group-btn:last-child>.btn,
 .input-group-btn:last-child>.btn-group {
   margin-left: 0px;
-}
\ No newline at end of file
+}
+
+.has-error {
+  .input-group-addon {
+    background-color: #f2dede;
```
Comment (bzmillerboy) [inline crossroads.net/styles/layout/forms.scss]: Should this use variable $state-danger-bg?

File: crossroads.net/styles/pages/give.scss [hunk 2]
```diff
@@ -14,8 +14,16 @@
         color: $gray-light;
         text-shadow: 0 1px 0 #ffffff;
     }
+
+    &.has-error {
```
Comment (bzmillerboy) [inline crossroads.net/styles/pages/give.scss]: Is this needed specifically for the give page or should the more global style take care of this?

File: crossroads.net/styles/layout/forms.scss [hunk 3]
```diff
@@ -37,7 +37,14 @@
...
-.input-group-btn:last-child>.btn, 
+.input-group-btn:last-child>.btn,
 .input-group-btn:last-child>.btn-group {
   margin-left: 0px;
-}
\ No newline at end of file
+}
+
+.has-error {
+  .input-group-addon {
```
Comment (bzmillerboy) [inline crossroads.net/styles/layout/forms.scss]: Can we add a text-shadow:none here? A text shadow of white with a red background doesn't work well. I'm curious why bootstraps defaults aren't taking hold here?

**Pull request merged**